### PR TITLE
feat: Add theme support 

### LINKED ISSUE #4: Theme support
`capture` is using [shiki](https://www.npmjs.com/package/shiki) for syntax highlighting. The default theme is `one-dark-pro` but there should be a way for the user to change the theme.

The best way is to have these settings in the vscode JSON.

### PR DESCRIPTION
**Pull request by coding-cosmos**
fixes #4

### CODE REVIEW

**Comment by coding-cosmos** [comment]
@khattakdev Please check this out .

**Comment by coding-cosmos** [comment]
@khattakdev I am new to git , so I don't understand how my previously merged commit "feat:  Show error on no selection" got included in this PR. That's what I have done:
1) I pushed the changes to my github fork from my local repo without realizing that I should sync my fork to include the changes.
2) Then I see there are merge conflicts so I discarded my commits from the github website itself
3) Then I sync the fork then I resolve the merge conflicts in my local repo and commit 
4) Then I push to my fork 
5) Created PR and saw previous commit is also included
Can you guide me ?

**Comment by khattakdev** [comment]
@coding-cosmos Don't worry about the commit history - as long as you have not modified the code.

File: package.json [hunk 1]
```diff
@@ -27,7 +27,18 @@
...
-    ]
+    ],
+     "configuration": [{
+        "title": "Capture settings",
+        "properties": {
+            "capture.themeName": {
+                "type": "string",
+                "default": "one-dark-pro",
+                "description": "The theme to use.",
+                "enum": ["dark-plus","dracula-soft","dracula","github-dark-dimmed","github-dark","github-light","hc-light","light-plus","material-theme-darker","material-theme-lighter","material-theme-ocean","material-theme","min-dark","min-light","monokai","nord","one-dark-pro","poimandres","rose-pine-dane","rose-pine","slack-dark","slack-ochin","solarized-dark","solarized-light","vitesse-dark","vitesse-light"]
```
Comment (khattakdev) [inline package.json:38]:
```suggestion
                "enum": ["dark-plus","dracula-soft","dracula","github-dark-dimmed","github-dark","github-light","hc_light","light-plus","material-theme-darker","material-theme-lighter","material-theme-ocean","material-theme","min-dark","min-light","monokai","nord","one-dark-pro","poimandres","rose-pine-dane","rose-pine","slack-dark","slack-ochin","solarized-dark","solarized-light","vitesse-dark","vitesse-light","material-theme-palenight", "rose-pine-dawn", "rose-pine-moon", "css-variables"]
```

Comment (coding-cosmos) [reply package.json]: I have added suggested changes.

File: package.json [hunk 2]
```diff
@@ -27,7 +27,18 @@
...
       }
-    ]
+    ],
+     "configuration": [{
+        "title": "Capture settings",
+        "properties": {
+            "capture.themeName": {
+                "type": "string",
+                "default": "one-dark-pro",
+                "description": "The theme to use.",
```
Comment (khattakdev) [inline package.json:37]:
```suggestion
                "description": "Select a theme for the screenshot. This theme will be used to style the code in the screenshot, making it look like it does in the VS Code editor with that theme applied.",
```

File: src/template.ts [hunk 3]
```diff
@@ -1,23 +1,29 @@
...
   });
 
   const htmlContent = highlighter.codeToHtml(content, "javascript");
   return await htmlContent;
 };
 
-const generateTemplate = async (content: string) => {
+const generateTemplate = async (content: string, themeName: string) => {
+  // get background color from the theme.
+  const highlighter = await shiki.getHighlighter({
```
Comment (khattakdev) [inline src/template.ts:14]: This function is used twice - here and in the `highlightCode`.

return `highlighter` from `highlightCode` and do rest of the things (`codeToHtml()` and `getBackgroundColor()`) in this function.

File: src/template.ts [hunk 4]
```diff
@@ -1,23 +1,29 @@
...
-const highlightCode = async (content: string) => {
+const highlightCode = async (themeName: string) => {
   const highlighter = await shiki.getHighlighter({
-    theme: "one-dark-pro", // You can specify other themes like 'dark-plus', 'light-plus', etc.
+    theme: themeName, // You can specify other themes like 'dark-plus', 'light-plus', etc.
   });
 
-  const htmlContent = highlighter.codeToHtml(content, "javascript");
-  return await htmlContent;
+  return await highlighter;
```
Comment (khattakdev) [inline src/template.ts:8]:
```suggestion
  return highlighter;
```

**Pull request merged**